blocks/library/freeform/index.js

```diff
@@ -22,7 +20,8 @@ registerBlockType( 'core/freeform', {
 	attributes: {
 		content: {
 			type: 'string',
-			source: prop( 'innerHTML' ),
+			source: 'property',
```

Comment:
I know it's a direct port, but this should be converted to `source: 'html'`, here or otherwise.